### DISCUSSION TITLE
Symfony Local Web Server Local Network Broken On MacOS Sequoia?

### DISCUSSION THREAD

**Discussion by dylanelev8webstudio** [symfony/symfony#58961] [Q&A]
My development work flow involves a Wordpress application running on Docker communicating with a Symfony running with the Symfony Local Web Server.

After upgrading to MacOS 15 Sequoia, all applications now need to be whitelisted under Privacy & Security > Local Network or they cannot be accessed via my local IP. This means my Docker hosted Wordpress application cannot access my Symfony application via my local IP unless Symfony (or PHP, not sure) has been whitelisted.

The problem is, MacOS does not prompt me to whitelist Symfony when I start up the server and there's no way to manually add applications to this whitelist (that I know of).

My workflow worked in previous versions of MacOS and I confirmed they could be accessed through my local IP so I'm quite sure this new Local Network whitelist is the issue.

So, what am I missing? Has my workflow been quietly bad up to this point and now I'm suffering the consequences? Shouldn't the local web server be able to be accessed by devices on my local network? I have searched and searched and come up with nothing, so I'm really at a loss with how to proceed. Any help or suggestions are appreciated.

**Comment by CharlyPoppins** [comment]
Hello,

I managed to be access on "symfony server:start" remotly instances by disabling the firewall... (on Sequoia : Network -> Firewall | Réseau -> Coupe-feu)

Still looking for a better way.

PS : Before disabling complelty the firewall I tried to allow symfony binary in firewall settings :
`/opt/homebrew/bin/symfony` and `/Users/****/.symfony5/bin/symfony`

also with command line `/usr/libexec/ApplicationFirewall/socketfilterfw --add /opt/homebrew/bin/symfony`

maybe we need to allow another binary ?

**Comment by kevinpapst** [comment]
I opened an issue here: https://github.com/symfony-cli/symfony-cli/issues/577
